src/main/java/org/commcare/formplayer/services/ResponseMetaDataTracker.java

```diff
@@ -23,6 +27,11 @@ public boolean isAttemptRestore() {
     }
 
     public void setAttemptRestore(boolean attemptRestore) {
+        final Span span = GlobalTracer.get().activeSpan();
+        if (span != null && (span instanceof MutableSpan)) {
```

Comment:
Does this want to use a check similar to the one here https://github.com/dimagi/commcare-core/pull/1063/commits/fcdceca3c81eb4be9b529e45cf9e86b81a59a807 ?